lib/ansible/config/manager.py

```diff
@@ -248,6 +237,21 @@ def __init__(self, conf_file=None, defs_file=None):
 
         # update constants
         self.update_config_data()
+        try:
+            self.update_module_defaults_groups()
+        except Exception as e:
+            # Since this is a 2.7 preview feature, we want to have it fail as gracefully as possible when there are issues.
+            sys.stderr.write('Could not load module_defaults_groups: %s: %s\n\n' % (type(e).__name__, e))
+            self.module_defaults_groups = {}
+
+    def _read_config_yaml_file(self, yml_file):
+        yml_file = to_bytes(yml_file)
+        if os.path.exists(yml_file):
```

Comment:
We probably need to decide what relative paths are relative to.  This currently would be relative to the current working directory and the past few cve's have been related to not doing that.

Reply:
Options for relative path base could either be the location of ansible.cfg, or the location of the current playbook.

Reply:
Right now this will never receive a relative path, because we turned off user-specified configuration. So I think we can add a TODO and call it good.

Reply:
Agreed.